imap_processing/cdfutils/defaults.py

```diff
@@ -0,0 +1,53 @@
+from dataclasses import dataclass
```

Comment:
I suggest adding a module docstring here that describes the purpose of the module, and maybe some example usage (same for `global_base.py`).

Reply:
I agree with Matthew. If these are utils (the default), a description would be helpful.